docs/getting-started/usage/loading-a-mnemonic.en.md

```diff
@@ -54,44 +54,48 @@ Enter each word of your BIP-39 mnemonic one at a time. Krux will disable impossi
 <img src="../../../img/maixpy_amigo/load-mnemonic-via-numbers-word-300.png" align="right" class="amigo">
 
 ##### Decimal
-Enter each word of your BIP-39 mnemonic as a number (1-2048) one at a time. You can use [this list](https://github.com/bitcoin/bips/blob/master/bip-0039/english.txt) for reference.
+Enter each word of your BIP39 mnemonic as a number (1-2048) one at a time. You can use [this list](https://github.com/bitcoin/bips/blob/master/bip-0039/english.txt) for reference.
 
 ##### Hexadecimal and Octal
-You can also enter your BIP-39 mnemonic word's numbers (1-2048) in hexadecimal format, with values ranging from 0x1 to 0x800, or in octal format, with values ranging from 01 to 04000. This is useful with some metal plate backups that uses those formats.
+You can also enter your BIP39 mnemonic word's numbers (1-2048) in hexadecimal format, with values ranging from 0x1 to 0x800, or in octal format, with values ranging from 01 to 04000. This is useful with some metal plate backups that uses those formats.
 
 <div style="clear: both"></div>
 
 #### Tiny Seed (Bits)
 <img src="../../../img/maixpy_m5stickv/load-mnemonic-via-tinyseed-filled-250.png" align="right" class="m5stickv">
 <img src="../../../img/maixpy_amigo/load-mnemonic-via-tinyseed-filled-300.png" align="right" class="amigo">
 
-Enter the BIP-39 mnemonic word's numbers (1-2048) in binary format, toggling necessary bits to recreate each of the word's respective number. The last word will have checksum bits dynamically toggled while you fill the bits.
+Enter the BIP39 mnemonic word's numbers (1-2048) in binary format, toggling necessary bits to recreate each of the word's respective number. The last word will have checksum bits dynamically toggled while you fill the bits.
```

Comment:
Although not significant, it must be said that coin flipping using this method will limit words from 1-2047, thus disabling word 2048 will reduce entropy. In my rough calcs, total possibilities will be reduced in:
 ~0.6% with 12 words
 ~1,2% with 24 words

Reply:
After clarification from @jdlcdl , disregard the above. I would just add instead:
"In case you flip 11 tails in a row, just leave the 2048 bit marked"